src/PostgREST/Error.hs

```diff
@@ -140,9 +138,6 @@ instance JSON.ToJSON ApiRequestError where
     Nothing
     (Just $ JSON.String $ "Verify that '" <> resource <> "' is included in the 'select' query parameter.")
 
-  toJSON LimitNoOrderError = toJsonPgrstError
-    ApiRequestErrorCode09 "A 'limit' was applied without an explicit 'order'" Nothing (Just "Apply an 'order' using unique column(s)")
-
```

Comment:
This also may be related. It breaks [limited updates/deletes](https://postgrest.org/en/latest/references/api/tables_views.html#limited-update-delete) which need `order` to be present in the query string.